ontology/yaml/resources/entity_types/global.yaml

```diff
@@ -53,4 +53,11 @@ INCOMPLETE:
   id: "4665780478685478912"
   description: "A known incomplete type."
   is_abstract: true
+  
+##Global namespace for tanks##
+TANK:
```

Comment:
should conform to [BDNS](https://github.com/theodi/BDNS/blob/master/BDNS_Abbreviations_Register.csv) unless there is a good reason not to. think `TK` is appropriate for a generic tank.

Reply:
Replaced the TANK to TK in the global namespace.